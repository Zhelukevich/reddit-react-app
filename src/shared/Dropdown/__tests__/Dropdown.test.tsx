
// /** @jest-environment jsdom */

// import React from "react";
// import { shallow } from "enzyme";
// import { Dropdown } from "../Dropdown";


// import Enzyme from 'enzyme';
// import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
// Enzyme.configure({ adapter: new Adapter() });


// describe('Dropdown', () => {
//    test('should render', () => {
//       const wrapper = shallow(<Dropdown children={<div />} button={<button />} />);
//       expect(wrapper).toBeDefined();
//       console.log(window);
//       expect(wrapper.find('div.container').isEmptyRender()).toBeFalsy();
//    });

//    test('should render (snapshot)', () => {
//       const wrapper = shallow(<Dropdown children={<div />} button={<button />} />);

//       expect(wrapper).toMatchSnapshot();
//    });
// });
