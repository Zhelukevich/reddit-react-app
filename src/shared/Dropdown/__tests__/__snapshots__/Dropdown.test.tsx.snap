// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown should render (snapshot) 1`] = `
<div
  className="container"
>
  <div
    onClick={[Function]}
  >
    <button />
  </div>
</div>
`;
